app/components/round-resource.js

```diff
@@ -1,6 +1,20 @@
 import Ember from 'ember';
 
-export default Ember.Component.extend({
-  classNames: [ 'col-xs-12', 'well', 'round-resource' ],
+const { Component, computed } = Ember;
+
+export default Component.extend({
+  classNames: [ 'round-resource' ],
   round: null,
+
+  roundIcon: computed('round.users', function() {
+    if(this.get('round.users.length') > 1) {
```

Comment:
Consider onelining this: `return this.get('round.users.length') > 1 ? 'people' : 'person';`